spec/integrations/factory_prof_spec.rb

```diff
@@ -13,6 +13,17 @@
       expect(output).to match(/\s+16\s+8\s+(\d+\.\d{4}s\s+){3}user\n\s+10\s+6\s+\s+(\d+\.\d{4}s\s+){3}post/)
     end
 
+    specify "simple printer with threshold param", :aggregate_failures do
+      output = run_rspec("factory_prof", env: {"FPROF" => "1", "FPROF_THRESHOLD" => "11"})
+
+      expect(output).to include("FactoryProf enabled (simple mode)")
+
+      expect(output).to include("Factories usage")
+      expect(output).to match(/Total: 16\n\s+Total top-level: 8\n\s+Total time: \d{2}+:\d{2}\.\d{3} \(out of \d{2}+:\d{2}\.\d{3}\)\n\s+Total uniq factories: 1/)
```

Comment:
We should only hide less used factories from the output but totals still must show the actual data, i.e., here we should see "Total uniq factories: 2".

Thus, we need to update the exclusion logic (no `select!`, just skip when printing) and update the test to check that there is not match for skipped factories.

Reply:
done